gcs/bucket.py

```diff
@@ -486,6 +486,8 @@ def insert_notification(self, request, context):
         return notification
 
     def get_notification(self, notification_id, context):
+        if notification_id not in self.notifications:
+            testbench.error.notfound("Notification %s" % notification_id, context)
```

Comment:
Should we add this same check to `delete_notification()` below?

Reply:
Done. PTAL.